pipedream_solver/nsuperlink.py

```diff
@@ -1777,26 +1777,45 @@ def numba_c_ik(u_Ip1k, sigma_ik):
     return -np.maximum(-u_Ip1k, 0) * sigma_ik
 
 @njit(float64[:](float64[:], float64, float64[:], float64[:], float64[:], float64[:],
-                 float64[:], float64[:], float64[:], float64[:], boolean[:], float64[:], float64),
+                 float64[:], float64[:], float64[:], float64[:], boolean[:], float64[:], int64[:], float64),
       cache=True)
 def numba_b_ik(dx_ik, dt, n_ik, Q_ik_t, A_ik, R_ik,
-               A_c_ik, C_ik, a_ik, c_ik, ctrl, sigma_ik, g=9.81):
+               A_c_ik, C_ik, a_ik, c_ik, ctrl, sigma_ik, Sf_method, g=9.81):
     """
     Compute link coefficient 'b' for link i, superlink k.
     """
     # TODO: Clean up
     cond = A_ik > 0
```

Comment:
This `cond` is no longer needed here. This line should be deleted.

Reply:
I deleted the 'cond' code.